src/include/daos/pool.h

```diff
@@ -78,6 +78,9 @@
  */
 #define DAOS_POOL_GLOBAL_VERSION 3
 
+/** Default system name used in srv_pool.c */
+extern char daos_sysname[];
+
```

Comment:
Since `daos_sysname` is defined in `engine`, its declaration should be in daos_engine.h instead, no?

Reply:
im not sure that it matters? we only use it in the the server client pool operations, but i will move it, which would require though including daos_engine.h

Reply:
actually pool/srv_cli.c already includes that, so that actually works better. thanks ill update.